types/options.d.ts

```diff
@@ -45,17 +45,17 @@ export interface ComponentOptions<V extends Vue> {
   delimiters?: [string, string];
 }
 
-export interface FunctionalComponentOptions {
+export interface FunctionalComponentOptions<P, S> {
   props?: string[] | { [key: string]: PropOptions | Constructor | Constructor[] };
   functional: boolean;
-  render(this: never, createElement: CreateElement, context: RenderContext): VNode;
+  render(this: never, createElement: CreateElement, context: RenderContext<P, S>): VNode;
   name?: string;
 }
 
-export interface RenderContext {
-  props: any;
+export interface RenderContext<P, S> {
+  props: P;
   children: VNode[];
-  slots: any;
+  slots(): S;
```

Comment:
Why is this changed to method? In my understand, it should a property of an object.